.github/workflows/ci.yml

```diff
@@ -18,25 +22,100 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
-          python-version: 2.7
+          python-version: 3.8
 
       - name: Install Python dependencies
         run: |
           pip install --upgrade pip
-          pip install yadage
+          pip install --pre reana-client
 
-      - name: Validate workflow
-        run: yadage-validate workflow/yadage/workflow.yaml | grep -q 'workflow validates'
+      - name: Validate workflows
+        run: |
+          for file in $FILES; do reana-client validate --environments -f $file; done
```

Comment:
It would be ideal to use wildcards here, since the list of `reana` spec files could be changed in the future and this list then should be updated manually.
What if we do something like this: `for file in reana*.yaml; do ...` ? WDYT?

Reply:
Makes sense! Thanks!